docs/pages/machine-id/reference/github-actions.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: GitHub Actions
+description: Reference for GitHub Actions joining
+---
+
```

Comment:
We should include an opening paragraph explaining what will be described on this page.